manim/mobject/svg/svg_mobject.py

```diff
@@ -182,8 +182,7 @@ def generate_mobject(self) -> None:
         element_tree = ET.parse(file_path)
         new_tree = self.modify_xml_tree(element_tree)
         # Create a temporary svg file to dump modified svg to be parsed
-        root, ext = os.path.splitext(file_path)
-        modified_file_path = root + "_" + ext
+        modified_file_path = modified_file_path.with_stem(file_path.stem + "_")
```

Comment:
This currently causes tests to fail, you probably want something like 
```suggestion
        modified_file_path = file_path.with_stem(file_path.stem + "_")
```

Reply:
Apparently `with_stem` is new from 3.9, and the read-the-docs build runs on 3.8

Reply:
True, then it will have to be rewritten.